src/v/config/configuration.cc

```diff
@@ -556,6 +556,19 @@ configuration::configuration()
       "wasn't reached",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       1ms)
+  , fetch_read_strategy(
+      *this,
+      "fetch_read_strategy",
+      "The strategy used to fulfill fetch requests",
+      {.needs_restart = needs_restart::no,
+       .example = model::fetch_read_strategy_to_string(
+         model::fetch_read_strategy::non_polling),
+       .visibility = visibility::user},
```

Comment:
should be tunable I think

Reply:
Will look into it. This was how it's configured in the current fetch path.

Reply:
Did you look into it? I don't think there's any benefit to making it user visible, just means people will fiddle with it.

> This was how it's configured in the current fetch path.

I didn't get that part: this configuration is new right?

Reply:
No sorry, I don't know what I was thinking here. I thought you were commenting on a different part of the code for some reason. Will change the visibility to `tunnable`.